### PR TITLE
CRDCDH 1724

### PR DESCRIPTION
**Pull request by vshand11**
Updated based on structure changes in content.json file:
1.  Model reader.
2. Term reader for pv puller.

### CODE REVIEW

File: src/common/pv_term_reader.py [hunk 1]
```diff
@@ -68,24 +69,29 @@ def create_cde(self, data_common, version):
...
         try:
-            self.log.info('Reading propr file: {} ...'.format(model_props_file))
-            if model_props_file and '.' in model_props_file and model_props_file.split('.')[-1].lower() in YML_FILE_EXT:
-                properties = download_file_to_dict(model_props_file).get(PROP_DEFINITIONS)
-                if not properties:
-                    msg = f'Invalid model properties file: {model_props_file}!'
-                    self.log.error(msg)
-                    return False, None, msg
+            for model_props_file in model_props_files:
+                self.log.info('Reading propr file: {} ...'.format(model_props_file))
```
Comment (n2iw) [inline src/common/pv_term_reader.py:79]: Typo in **propr**

File: src/common/pv_term_reader.py [hunk 2]
```diff
@@ -48,14 +49,14 @@ def __init__(self, model_def_loc, tier):
     create a CDE term dict by parsing yaml model property file
     """
     def create_cde(self, data_common, version):
-        dc = data_common.upper()
+        dc = data_common
         v = self.models_def[dc]
         model_dir = os.path.join(self.model_def_dir, os.path.join(dc, version))
         #process model files for the data common
-        props_file_name = os.path.join(model_dir, v[DEF_MODEL_PROP_FILE])
+        props_file_names = [os.path.join(model_dir, file) for file in v[DEF_MODEL_FILES]]
```
Comment (n2iw) [inline src/common/pv_term_reader.py:56]: I suggest rename the variable to **model_file_names**

File: src/common/pv_term_reader.py [hunk 3]
```diff
@@ -48,14 +49,14 @@ def __init__(self, model_def_loc, tier):
...
         #process model files for the data common
         try:
-            result, properties_term, msg = self.parse_model_props(props_file_name)
+            result, properties_term, msg = self.parse_model_props(props_file_names)
```
Comment (n2iw) [inline src/common/pv_term_reader.py:59]: I suggest rename the variable to **model_file_names**

File: src/common/pv_term_reader.py [hunk 4]
```diff
@@ -68,24 +69,29 @@ def create_cde(self, data_common, version):
     """
     parse model property file
     """
-    def parse_model_props(self, model_props_file):
-        properties = None
+    def parse_model_props(self, model_props_files):
```
Comment (n2iw) [inline src/common/pv_term_reader.py:72]: I suggest rename the variable to **model_file_names**

File: src/common/pv_term_reader.py [hunk 5]
```diff
@@ -68,24 +69,29 @@ def create_cde(self, data_common, version):
...
+                self.log.error(msg)
+                return False, None, msg
+            
         except Exception as e:
-            self.log.exception(e)
-            msg = f'Failed to read yaml file to dict: {model_props_file}!'
-            self.log.exception(msg)
-            raise e
+                    self.log.exception(e)
+                    msg = f'Failed to read yaml file to dict: {model_props_file}!'
```
Comment (n2iw) [inline src/common/pv_term_reader.py:92]: I think **model_props_file** needs to be updated also

File: src/common/pv_term_reader.py [hunk 6]
```diff
@@ -68,24 +69,29 @@ def create_cde(self, data_common, version):
...
+                msg = f'Invalid model properties file: {model_props_file}!'
+                self.log.error(msg)
+                return False, None, msg
+            
         except Exception as e:
-            self.log.exception(e)
-            msg = f'Failed to read yaml file to dict: {model_props_file}!'
-            self.log.exception(msg)
-            raise e
+                    self.log.exception(e)
```
Comment (n2iw) [inline src/common/pv_term_reader.py:91]: The indentation seems not correct (two levels in?)

**Review by n2iw — CHANGES_REQUESTED** [review]

**Comment by vshand11** [comment]
Updated, please continue.

**Review by n2iw — APPROVED** [review]
LGTM

**Pull request merged**